projects/ff4j-properties/src/app/properties.module.ts

```diff
@@ -0,0 +1,27 @@
+import {Injector, NgModule} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PropertiesComponent } from './properties.component';
+import { environment } from '../environments/environment';
+import {createCustomElement} from '@angular/elements';
+
+
+@NgModule({
+  declarations: [PropertiesComponent],
+  imports: [
+    CommonModule
```

Comment:
Did you try hosting the sub project independently? The **BrowserModule** is missing and hence hosting it separately would not have worked

Reply:
yes @J-DK, Thank you :)
I missed to host the sub project independently